syntax = "proto3";

package holamundo;

// Definici√≥n del servicio
service HolaMundoService {
    rpc DecirHola (HolaRequest) returns (HolaResponse);
}

// Mensaje de solicitud
message HolaRequest {
    string nombre = 1;
}

// Mensaje de respuesta
message HolaResponse {
    string mensaje = 1;
}
